src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const createNotification = (text, cssClass = '') =>
+  `<div class="message ${cssClass}">${text}</div>`;
+
+const promise1 = new Promise((resolve) => resolve('Promise was resolved!'));
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => reject(new Error('Promise was rejected!')), 3000);
```

Comment:
Bug: This promise will always be rejected after 3 seconds. This might not be what you want. Consider moving the `setTimeout` into the `addEventListener` function.